e2e/provision/playbooks/roles/bootstrap/defaults/main.yml

```diff
@@ -101,6 +102,11 @@ cluster_api:
         version: "{{ nephio_catalog_revision }}"
         async: 1500
         poll: 0
+      - repo_uri: "{{ nephio_catalog_repo_uri }}"
+        pkg: infra/capi/cluster-capi-infrastructure-metal3
```

Comment:
Does it matter that the file is now "cluster-api-infrastructure-metal3.yaml" instead of "cluster-capi-infrastructure-metal3.yaml"

Reply:
@dkosteck good catch. I will change the file name and push an update.

Reply:
Well, the `infra/capi/cluster-capi-infrastructure-metal3` is the name of the package directory, so I think it is good as is.